.github/workflows/rdev-update-for-pr.yml

```diff
@@ -2,9 +2,9 @@ name: Create/Update rdev for PR
 
 on:
   pull_request:
-    branches-ignore:
-      - prod
-      - "renovate/*"
+    branches:
```

Comment:
from the docs: If you define a branch with the ! character, you must also define at least one branch without the ! character. If you only want to exclude branches, use branches-ignore instead.

Reply:
Oh damnit.